content/security/for-admins/access-tokens.md

```diff
@@ -0,0 +1,114 @@
+  personal account's limits.
+
+If you have existing [service accounts](/docker-hub/service-accounts/), Docker recommends that you replace the service accounts with OATs. OATs offer the following advantages over service accounts:
+
+- Access permissions are easier to manage with OATs. You can assign access
+  permissions to OATs, while service accounts require using teams for access
+  permissions.
+- OATs are easier to manage. OATs are centrally managed in the Admin Console.
+  For service accounts, you may need to sign in to that service account to
+  manage it. If using single sign-on enforcement and the service account is not
+  in your IdP, you may not be able to sign in to the service account to manage
+  it.
+- OATs are not associated with a single user. If a user with access to the
+  service account leaves your organization, you may lose access to the service
+  account. OATs can be managed by any organization owner.
+
+## Create an organization access token
+
+> [!IMPORTANT]
+>
+> Treat access tokens like a password and keep them secret. Store your tokens securely in a credential manager for example.
+
+Organization owners can create up to 3 organization access tokens (OATs) for
+organizations with a Team subscription and up to 100 OATs for organizations with
+a Business subscription. Expired tokens count towards the total amount of
+tokens.
+
+To create an OAT:
+
+1. Sign in to the [Admin Console](https://app.docker.com/admin).
+
+2. Select the organization you want to create an access token for.
+
+3. Under **Security and access**, select **Access tokens**.
+
+4. Select **Generate access token**.
+
+5. Add a label and optional description for your token. Use something that indicates the use case or purpose of the token.
+
+6. Select the expiration date for the token.
+
+7. Select the repository access for the token.
+
+   The access permissions are scopes that set restrictions in your repositories.
+   For example, for Read & Write permissions, an automation pipeline can build
+   an image and then push it to a repository. However, it can't delete the
+   repository. You can select one of the following options:
+
+   - **Public repositories (read only)**
+   - **All repositories**: You can select read access, or read and write access.
+   - **Select repositories**: You can select up to 50 repositories, and then
+     select read access, or read and write access for each repository.
+
+8. Select **Generate token** and then copy the token that appears on the screen
+   and save it. You won't be able to retrieve the token once you exit the
+   screen.
+
+## Use an organization access token
+
+You can use an organization access token when you sign in using Docker CLI.
+
+Sign in from your Docker CLI client with the following command, replacing
+`YOUR_ORG` with your organization name:
+
+```console
+$ docker login --username <YOUR_ORG>
+```
+
+When prompted for a password, enter your organization access token instead of a
+password.
```

Comment:
**suggestion (non-blocking):** Maybe we can add another example of using this in CI (probably a Github action example).

Reply:
I'm not sure we need any other example right now. It's the same command as before, just the orgname and oat being used.